portal/src/graphql/adminapi/AuditLogEntryScreen.tsx

```diff
@@ -190,6 +190,8 @@ const AuditLogEntryScreen: React.VFC = function AuditLogEntryScreen() {
                 wrappingIndent: "deepIndent",
                 renderLineHighlight: "none",
               }}
+              isCopyButtonVisible={true}
+              copyValue={code}
```

Comment:
Introduce a new component to represent a code block. Do not make changes to CodeEditor.

```
// The implementation use monaco editor under the hood.
// It supports copy button out-of-the-box.
interface CodeBlockProps {
  className?: string;
  value?: string;
  language?: string;
}
```